docs/streams/developer-guide/dsl-api.html

```diff
@@ -261,7 +261,7 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
                         <p>You <strong>must specify SerDes explicitly</strong> if the key or value types of the records in the Kafka input
                             topics do not match the configured default SerDes. For information about configuring default SerDes, available
                             SerDes, and implementing your own custom SerDes see <a class="reference internal" href="datatypes.html#streams-developer-guide-serdes"><span class="std std-ref">Data Types and Serialization</span></a>.</p>
-                        <p class="last">Several variants of <code class="docutils literal"><span class="pre">stream</span></code> exist, for example to specify a regex pattern for input topics to read from).</p>
+                        <p class="last">Several variants of <code class="docutils literal"><span class="pre">stream</span></code> exist. For example, you can specify a regex pattern for input topics to read from (note that a single nodegroup will be created for all matching topics, and therefore the number of tasks will scale with the maximum partition count rather than the total number of partitions).</p>
```

Comment:
Is `nodegroup` already a defined term by this point? Looking at this and the other changes, this might be kind of a confusing usage of the word "node".  Maybe just make up something more intuitive, like "input topic group"?

Reply:
Yeah, that's fair. I didn't know what that was when I first saw it in the code. What do you think about using "subtopology" here?

Alternatively, how about: `note that all matching topics will be part of the same input topic group, and the work will not be parallelized for different topics if subscribed to in this way`

Reply:
I think your "alternative" is the best phrasing yet.

Reply:
Done. Rephrased the javadocs as well -- thanks!